HAL/src/eoss3_hal_i2c.c

```diff
@@ -23,12 +26,12 @@
 #include <stdint.h>
 #include <string.h>
 #include <stddef.h>
-#include <eoss3_dev.h>
-#include <eoss3_hal_def.h>
-#include <eoss3_hal_i2c.h>
-#include <eoss3_hal_pad_config.h>
+#include "eoss3_dev.h"
+#include "eoss3_hal_def.h"
+#include "eoss3_hal_i2c.h"
+#include "eoss3_hal_pad_config.h"
 #include "test_types.h"
-#include <eoss3_hal_wb.h>
+#include "eoss3_hal_wb.h"

```

Comment:
Please don't refactor the original code - we want to keep it as similar to the original code as possible.